src/index.html

```diff
@@ -16,7 +16,11 @@
       href="./styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Stop watch</h1>
+  <body class="page">
+    <div class="stopwatch stopwatch--speed-up">
```

Comment:
The BEM modifier `stopwatch--speed-up` seems to be used without the base class `stopwatch`. According to BEM methodology, you should always include the base block class when using modifiers.